app/models/license.rb

```diff
@@ -501,6 +508,28 @@ def permissions
 
   def entitlement_codes = entitlements.reorder(nil).codes
   def entitlement_ids   = entitlements.reorder(nil).ids
+  def user_ids          = users.reorder(nil).ids
+
+  # NOTE(ezekg) For backwards compatibility. Dual writing until we migrate
+  #             data from licenses.user_id column to HABTM.
+  def user=(record)
+    case record
+    in String => user_id
+      self.users = User.where(id: user_id)
```

Comment:
Needs to be account scoped.

Reply:
For time being, we could duplicate the `account_id`  attribute during license creation to make sure it's assigned first.

```rb
current_account.licenses.create(
  account_id: current_account.id,   
  **license_params,
)
```